force-app/main/default/lwc/dxTemplateSetup/dxTemplateSetup.js

```diff
@@ -5,20 +5,82 @@ import getRelatedToTypeOptions from '@salesforce/apex/SaveDocumentTemplate.getRe
+import createLog from '@salesforce/apex/LogHandler.createLog';
 
 export default class DxTemplateSetup extends LightningElement {
+  //variables added by Bhavya for watermark starts here
+  step = 1; // progress bar increases with a step value 1
+  currentStep = "1"; // It reflects the current (or) active step in the progress bar
+  baseURL; // domain base url of the ORG
+  imageUrl; // it stores the data of the uploaded image
+  @track activeTab =''; // It stores the active tab value
+  @track outerContainer = ''; //The styling of the container which holds the canvas
+  @track watermarkText  = ''; // The input text value with which text watermark is being created
+  @track showwatermarkbtn = false; // This variable controls the display of the Watermark Popup in HTML
+  @track fontSizeValue  =  '22'; // Default Font Size = 22
+  @track rotationValue = '0'; // Default Text Rotation Value = 0
+  @track rotationImagevalue = '0'; // Default Image Rotation Value = 0
+  previousRotationValue = '0'; // This variable stores the previous text rotation value, used for comparing with the current userinput text rotation value
+  previousImgRotationValue = '0'; // This variable stores the previous image rotation value, used for comparing with the current userinput image rotation value
+  @track colorValue = '#000000'; // Default watermark text color #000000
+  @track opacityImageValue = '1.0'; // Default Image Opacity = 1
+  @track checkedValText = true; // Default text watermark checkbox (isPrimary) is checked, true
+  @track checkedValImage = false; // Default Image watermark checkbox (isPrimary) is unchecked, false
+  @track imageScalingValue = 100; // Default Image scaling = 100
+  @track opacityValue = '1.0'; // Default text Opacity = 1
+  @track pageTextOption = 'All Pages - Text'; // Default text page Option for watermark is selected as "ALL PAGES"
+  @track pageImageOption = 'All Pages - Image'; // Default Image page Option for watermark is selected as "ALL PAGES"
+  //watermarkPageOptionsText combobox options for Text Watermark
+  watermarkPageOptionsText = [
+        { label: 'All Pages', value: 'All Pages - Text', checked : true },
+        { label: 'All Pages Except First Page', value: 'All Pages Except First Page - Text' , checked : false},
+    ];
+  //watermarkPageOptionsImage combobox options for Image Watermark
+  watermarkPageOptionsImage = [
+        { label: 'All Pages', value: 'All Pages - Image' , checked : true},
+        { label: 'All Pages Except First Page', value: 'All Pages Except First Page - Image', checked : false },
+    ];
+  //fontSizeOptions combobox options for Text Watermark
+ get fontSizeOptions() {
+    const options = [];
+    for (let i = 8; i <= 28; i += 2) {
+        options.push({
+            label: i.toString(),
+            value: i.toString()
+        });
+    }
+    return options;
+}
+  //opacityOptions combobox options for both Text & Image Watermark
+    get opacityOptions() {
```

Comment:
Can we get the opacity options corrected to Percentage instead of decimal with 0.1 as step, that will be be more user friendly.